tools/modules/EnsEMBL/Web/Component/Tools/VR/InputForm.pm

```diff
@@ -0,0 +1,280 @@
+=head1 LICENSE
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2018] EMBL-European Bioinformatics Institute
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=cut
+
+package EnsEMBL::Web::Component::Tools::VR::InputForm;
+
+use strict;
+use warnings;
+
+use EnsEMBL::Web::VRConstants qw(INPUT_FORMATS);
+
+use parent qw(
+  EnsEMBL::Web::Component::Tools::VR
+  EnsEMBL::Web::Component::Tools::InputForm
+);
+
+sub form_header_info {
+  ## Abstract method implementation
+  my $self = shift;
+
+  return $self->tool_header({'reset' => 'Clear form', 'cancel' => 'Close'});
+}
+
+sub get_cacheable_form_node {
+  ## Abstract method implementation
+  my $self            = shift;
+  my $hub             = $self->hub;
+  my $object          = $self->object;
+  my $sd              = $hub->species_defs;
+  my $species         = $object->species_list;
+  my $form            = $self->new_tool_form;
+  my $fd              = $object->get_form_details;
+  my $input_formats   = INPUT_FORMATS;
+  my $input_fieldset  = $form->add_fieldset({'no_required_notes' => 1});
+  my $current_species = $self->current_species;
+  my $msg             = $self->species_specific_info($self->current_species, 'VR', 'VR', 1);
+
+  my ($current_species_data)  = grep { $_->{value} eq $current_species } @$species;
+  my @available_input_formats = grep { $current_species_data->{example}->{$_->{value}} } @$input_formats;
+
+
+  # Species dropdown list with stt classes to dynamically toggle other fields
+  $input_fieldset->add_field({
+    'label'         => 'Species',
+    'elements'      => [{
+      'type'          => 'speciesdropdown',
+      'name'          => 'species',
+      'values'        => [ map {
+        'value'         => $_->{'value'},
+        'caption'       => $_->{'caption'},
+        'class'         => [  #selectToToggle classes for JavaScript
```

Comment:
Does this condition applies for VR as well? So its trying to only show species with variation and refseq